libs/designsystem/src/lib/components/dropdown/dropdown.component.scss

```diff
@@ -1,14 +1,18 @@
 @use '~@kirbydesign/core/src/scss/utils';
+@use '~@kirbydesign/core/src/scss/interaction-state';
```

Comment:
I suggest that we order imports (`@use` and `@forward`) alphabetically, unless there's a technical reason for not doing so.

It's just a code convention thing and not very important. It may however save us a few merge conflicts when we merge this with the hover interaction state PR 😎